acceptance/bundle/templates/wrong-url/script

```diff
@@ -0,0 +1,2 @@
+export NO_COLOR=1
```

Comment:
nit: setting `TERM=dumb` captures the intent better. That's what we do in other tests.